testdata/scripts/func-newlines.txt

```diff
@@ -469,3 +485,19 @@ func f(p1 string,
 	println("body")
 	return 0
 }
+
+func a() {
+	f := func(s string,
+		b bool,
+	) {
+		// foo
+	}
+}
+
+func f(p1 string,
+	p2 string) (int, string,
+
+/* baz */) {
```

Comment:
Hmm, is the "golden" version here better? The original code was weird to begin with for sure, but the new code feels worse. We moved the empty line for no good reason.

Reply:
An open question but not meaning to block the merge.